plugins/repository-s3/build.gradle

```diff
@@ -331,7 +331,16 @@ thirdPartyAudit.ignoreMissingClasses (
   'software.amazon.ion.system.IonBinaryWriterBuilder',
   'software.amazon.ion.system.IonSystemBuilder',
   'software.amazon.ion.system.IonTextWriterBuilder',
-  'software.amazon.ion.system.IonWriterBuilder'
+  'software.amazon.ion.system.IonWriterBuilder',
+  // We don't use the kms dependency 
```

Comment:
So I guess something else on the classpath references these classes?

Reply:
Yes, probably the other AWS jars that are on the cp.